server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/cache/CacheBlobStore.java

```diff
@@ -40,19 +40,19 @@
 
 import reactor.core.publisher.Mono;
 
-public class CachedBlobStore implements BlobStore {
+public class CacheBlobStore implements BlobStore {
```

Comment:
Why not keeping `CachedBlobStore` for the name here? I found it less confusing than having `BlobStoreCache` and `CacheBlobStore` :)

Reply:
ok i will change it back